Cmdr/BuiltInCommands/Tree/cd.lua

```diff
@@ -0,0 +1,71 @@
+local AncestryChanged;
+
+return {
+	Name = "cd",
+	Description = "Changes the current working instance to the specified instance.",
+	Group = "DefaultTree",
+	Args = {
+		{
+			Type = "string",
+			Name = "Instance",
+			Description = "The instance to change to. Can be a relative path or a full path."
+		}
+	},
+	AutoExec={
+		"cd game"
+	},
+	ClientRun = function(context, stringPath)
+		local util = context.Cmdr.Util
+		local treeView = context:GetStore("_TreeView")
+
+		--[[ Initializing state ]]--
+		if treeView.View == nil then
+			treeView.View = "Client"
+		end
+
+		--[[ Changing active instance ]]--
+		if treeView.View == "Client" then
+			local Instance;
+			
+			if string.split(stringPath,".")[1] == "game" then -- Absolute path specified
+				Instance = util.GetInstanceFromStringPath(stringPath)
+			else -- Relative path specified
+				local StartingInstance;
+				if not game:IsAncestorOf(treeView.WorkingInstance) and treeView.WorkingInstance ~= game then --Instance is in a detached state from the datamodel
+					context:Reply("The current working instance has been destroyed, unexpected behavior may occur!\n")
```

Comment:
Detached state doesn't necessarily mean destroyed; for example, character models are removed when the player dies but they do not get destroyed. I would change message to:

```suggestion
					context:Reply("Warning: the current working instance has been detached from the Data Model\n")
```

Reply:
Fixed in https://github.com/evaera/Cmdr/pull/98/commits/93a49f5f0dbb977604498dd57f86aed92381deb0 and https://github.com/evaera/Cmdr/pull/98/commits/65b56b26fd806038216f8f6da3a8826d9cb75622